ga4gh/backend.py

```diff
@@ -413,6 +429,18 @@ def getVariantSet(self, id_):
         dataset = self._getDatasetFromCompoundId(id_)
         return self.runGetRequest(dataset.getVariantSetIdMap(), id_)
 
+    def getVariant(self, id_):
+        """
+        Returns a variant with the given id
+        """
+        compoundId = variants.CompoundVariantId(id_)
+        dataset = self._getDatasetFromCompoundId(compoundId.datasetId)
+        variantSet = _safeMapQuery(
```

Comment:
OK, reading through the rest of the code there would be a lot of work to move over to an getter based approach. Let's do it this way for now, and try to refactor away from all this passing around map references at some point.